corehq/apps/userreports/tests/test_report_aggregation.py

```diff
@@ -533,7 +583,7 @@ def test_total_row_with_expanded_column(self):
                     ],
                     [3, 1, 0, 3],
                     [6, 0, 2, 2],
-                    [9, 1, 2, 5],
```

Comment:
Whenever you have a `min` column SQL reports will just sum up all of the mins, which seems pretty useless to me https://github.com/dimagi/sql-agg/blob/c1b70052fcbcfda219ac5532fa0ffcb87d84594c/sqlagg/base.py#L86-L88

Noted it as a bug, but it's been around for a while so doubt it's high priority. ES handles this properly